pkg/util/fileutil/filesystem.go

```diff
@@ -32,7 +33,17 @@ var (
 func GetMountedFiles(targetPath string) (map[string]string, error) {
 	paths := make(map[string]string)
 	// loop thru all the mounted files
-	files, err := os.ReadDir(targetPath)
+	var files []string
+	err := filepath.Walk(targetPath, func(path string, fi os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if fi.IsDir() {
+			return nil
+		}
+		files = append(files, strings.TrimPrefix(path, targetPath+string(os.PathSeparator)))
```

Comment:
What do you think about using [`filepath.Rel`](https://golang.org/pkg/path/filepath/#Rel`) to get the relative path instead of the strings prefix?

Reply:
yeah, this is better :+1: